benchmarks/experimental_ampnet.py

```diff
@@ -518,6 +613,7 @@ def bench_mpi(args):
 parser.add_argument("--max-batch", type=int, default=4, help="Max number of batches")
 parser.add_argument("--socket-name", type=str, default=None, help="socket ifname for gloo/tp")
 parser.add_argument("--num-decoder-layers", type=int, default=10, help="Number of decoder layers in the model")
+parser.add_argument("--spectrain", action="store_true", default=False, help="Use spectrain based weight prediction")
```

Comment:
Do we want to enable these benchmarks in circleCI?

Reply:
I think we can do that later.